parser/src/main/scala/hmda/parser/fi/FIDataCsvParser.scala

```diff
@@ -13,8 +13,9 @@ class FIDataCsvParser extends FIDataParser[String] {
   def parseLines(lines: Iterable[String]): FIData = {
     val tsLine = lines.head
     val ts = TsCsvParser(tsLine)
-    val lars = lines.tail.map(l => LarCsvParser(l))
-    FIData(ts, lars)
+    // Only parse LARs that do not contain errors
+    val larsWithoutErrors = lines.tail.map(l => LarCsvParser(l)).filter(_.isRight).map(_.right.get)
+    FIData(ts, larsWithoutErrors)
```

Comment:
That idea might have potential, but regardless I think it's outside the scope of this ticket (the change in this file was just to address the different return type of `LarCsvParser`).  Maybe make this an issue where it can be discussed more?

Also, I can't find where this class is being used, so I can't say for certain whether the collection of failures or error-free LARs would be more appropriate.